components/http_kernel.rst

```diff
@@ -545,6 +545,11 @@ below for more details).
     When setting a response for the ``kernel.exception`` event, the propagation
     is stopped. This means listeners with lower priority won't be executed.
 
+.. tip::
+
+    When setting a response for the ``kernel.exception`` event in order for the 
+    HttpKernel to use the Status Code of the response the :method:`Symfony\\Component\\HttpKernel\\Event\\GetResponseForExceptionEvent::allowCustomResponseCode` must be called inside your listener.
```

Comment:
`HttpKernel` should probably be in double backticks, sorry didn't notice it the first time.

Reply:
>  should probably 

are you sure? Checking on the same document other links have this form. Meaning with single quotes.

Reply:
Let's wait for someone else to reply